Makefile

```diff
@@ -188,10 +202,14 @@ test: clean linkcheck spellcheck  ## Run linkcheck, spellcheck
 deploy: clean html
 
 .PHONY: livehtml
-livehtml:  ## Rebuild Sphinx documentation on changes, with live-reload in the browser
-	cd "$(DOCS_DIR)" && sphinx-autobuild \
+livehtml: deps		## Rebuild Sphinx documentation on changes, with live-reload in the browser
+	cd "$(DOCS_DIR)" && ${SPHINXAUTOBUILD} \
 		--ignore "*.swp" \
 		-b html . "$(BUILDDIR)/html" $(SPHINXOPTS) $(O)
 
 .PHONY: all
 all: clean spellcheck linkcheck html ## Run checks and build html
+
+.PHONY: serve-docs
+serve-docs: html		## Start an HTTP server for docs
+	python -m http.server --directory ./_build/html
```

Comment:
I think this target is redundant to `livehtml`. I don't think we need this, unless I am missing something.

Reply:
I've added it as a way to test the generated static html file. It was unclear to me how Sphinx generates the toctree based navigation and why it missed, sometimes, the section that I was adding. I can remove it, but I find it reassuring to have an easy way to "test the final build".

Reply:
Fixed in 3ee76c9f93654dbbe9505b70c6054ccf9ca30797